index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/83eb6c852a.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./css/index.css">
+  <title>Product Management Workshop</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="index.html" class="logo">PM Global <br />2022</a>
+      <div id="hamburger">
+        <div class="ham-bar"></div>
+        <div class="ham-bar"></div>
+        <div class="ham-bar"></div>
+      </div>
+      <div class="menu">
+        <div class="menu-element close">
+          <i class="fa fa-times fa-1x"></i>
+        </div>
```

Comment:
- [ ] The top dark bar is missing from your navigation design. Please reference the design template and make the appropriate changes 🙏 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/164912015-5947b1c4-ba73-4832-a751-b1ff8ad2ecf6.png)
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/164912043-41061e61-7966-416b-86be-ae60138f1732.png)